apps/files_versions/src/css/versions.css

```
@@ -68,3 +68,7 @@
 	float: right;
 	margin-right: -10px;
 }
+
+.versionsTabView .emptycontent {
+	margin-top: 10px !important;
```

I would argue that 50px look better :)
```suggestion
	margin-top: 50px !important;
```